packages/application-shell/src/utils/http-client.ts

```diff
@@ -39,6 +41,10 @@ export type TForwardToConfig = {
    * The audience policy for verifying the incoming request from the Merchant Center API.
    */
   audiencePolicy?: TForwardToAudiencePolicy;
+  /**
+   * The claims to be included in the exchange JWT sent to the external API.
+   */
+  exchangeTokenClaims?: TForwardToExchangeTokenClaim[];
```

Comment:
I feel like the name might be a bit confusing and inconsistent with other options. For example the `audiencePolicy` is also technically a claim but we named it differently to better express its meaning.

How about something like `additionalFeatures`?

On the other hand I'm also wondering if we should expose the new claims as individual options. So for `permissions` it would be something like `includeUserPermissions`.

```js
{
  includeUserPermissions: true,
}
```

We can then follow the naming pattern `includeX` for other features. Internally we map those to the HTTP header `X-Forward-To-Claims`.

What do you think?

Reply:
I tried to use a name very specific to what the prop will be used for, but maybe it's just meaningful to me because I know about the inners of this topic and it will not be as clear for our users.

So I guess your proposal is either using `additionalFeatures` or `includeX`, right?
Personally I find `additionalFeatures` a little abstract and I prefer the `includeX` pattern, although we need to take into account that, if we choose it, we will need to release a new `app-kit` version every time we want to use a new "claim". This might not be a big problem since we would need to do it anyway because of the express auth middleware (reading the new claim from the token and expose it in the request).

Reply:
Yes for every new feature claim we would need to release a new version anyway.
This is actually good in the sense that in the documentation we can clearly state from which version onwards that particular feature is available.

Then let's try with the `includeX` pattern?

Reply:
Updated in this commit: 31ecb19b93fb2e4c6985ea070ad24382d33b348b